src/BusinessLayer/TelemetryReporting.cpp

```diff
@@ -142,7 +153,7 @@ void TelemetryReporting::sendCmuData(unsigned char cmuDataIndex)
    sendData(packet, packetLength);
 }
 
-void TelemetryReporting::sendMpptData(unsigned char mpptDataIndex)
+void TelemetryReporting::sendMppt(unsigned char mpptDataIndex)

```

Comment:
Added a method called SendMppt that is called by SendMpptData multiple times. The old method was to have a for loop and call SendMpptData from the View, the only functional change was that the method in View was moved to TelemetryReporting. Thoughts on passing in an int and just doing the loop in the method itself?